gen/python/sym/__init__.py

```diff
@@ -1,21 +1,8 @@
 # -----------------------------------------------------------------------------
 # This file was autogenerated by symforce from template:
-#     geo_package/__init__.py.jinja
+#     function/namespace_init.py.jinja
 # Do NOT modify by hand.
 # -----------------------------------------------------------------------------
 
-"""
-Python runtime geometry package.
-"""
-from .atan_camera_cal import ATANCameraCal
-from .double_sphere_camera_cal import DoubleSphereCameraCal
-from .equirectangular_camera_cal import EquirectangularCameraCal
-from .linear_camera_cal import LinearCameraCal
-from .polynomial_camera_cal import PolynomialCameraCal
-from .pose2 import Pose2
-from .pose3 import Pose3
-from .rot2 import Rot2
-from .rot3 import Rot3
-from .spherical_camera_cal import SphericalCameraCal
-
-epsilon = 2.220446049250313e-15
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore  # mypy issue #1422
+from ._init import *
```

Comment:
It's probably worth a comment either here or in `sym/_init.py` with some mention of what this is doing

Reply:
Okay. I added a comment. Let me know what you think.

Reply:
How does this sound?
``` python3
# Re-export core geometry and camera types
from ._init import *
```

Reply:
I was imagining something describing why we do this as opposed to what it's doing (since generally what it's doing should be clear from the code :) ), something like:
```python
# Import `sym` types onto the top-level module.  Needs to be in a separate `_init` module since
# namespace packages do not evaluate every copy of `__init__.py` , just the first discovered
```

Reply:
Sounds good. I'll adjust accordingly.

Reply:
As a preview, I am going with
``` python3
# Import sym types into the top level of the package. Needs to be done in all portions of
# the package since namespace packages do not evaluate every copy of `__init__.py`, just
# the first discovered. Import must occur after __path__ has been modified to ensure the
# sym._init module can be located.
```

Just a note, since the only files we are importing are the geo and cam cal classes, it's not necessary to have the import be from `_init.py` (this was only necessary if any portion of the package could add imports to the `__init__.py`). We could just do `from .rot3 import Rot3` and the like directly.

Reply:
Yeah I do think it might make sense to just add them back in here directly, although it might be a little confusing if other generated `sym` modules also have those imports?  Not sure, I'm fine either way

Reply:
I could also go either way. Though, since it is already `from ._init import *`, absent a compelling argument otherwise, I'm inclined to keep it that way.